File: app/src/main/java/org/michaelbel/moviemade/ui/modules/about/AboutActivity.java
package org.michaelbel.moviemade.ui.modules.about;

import android.os.Bundle;

import org.michaelbel.moviemade.R;
import org.michaelbel.moviemade.ui.base.BaseActivity;
import org.michaelbel.moviemade.ui.modules.about.fragments.AboutFragment;

import androidx.annotation.Nullable;
import androidx.appcompat.widget.AppCompatTextView;
import androidx.appcompat.widget.Toolbar;
import butterknife.BindView;

public class AboutActivity extends BaseActivity {

    @BindView(R.id.toolbar) Toolbar toolbar;
    @BindView(R.id.toolbar_title) AppCompatTextView toolbarTitle;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setSupportActionBar(toolbar);
        setContentView(R.layout.activity_about);

        if (savedInstanceState == null) {
            startFragment(new AboutFragment(), R.id.fragment_view);
        }
    }

    public Toolbar getToolbar() {
        return toolbar;
    }

    public AppCompatTextView getToolbarTitle() {
        return toolbarTitle;
    }
}

File: app/src/main/java/org/michaelbel/moviemade/ui/modules/about/AboutActivity.kt
package org.michaelbel.moviemade.ui.modules.about

import android.os.Bundle
import androidx.appcompat.widget.AppCompatTextView
import androidx.appcompat.widget.Toolbar
import kotlinx.android.synthetic.main.activity_title.*
import org.michaelbel.moviemade.R
import org.michaelbel.moviemade.ui.base.BaseActivity
import org.michaelbel.moviemade.ui.modules.about.fragments.AboutFragment

class AboutActivity : BaseActivity() {

    fun getToolbar() : Toolbar {
        return toolbar
    }

    fun getToolbarTitle() : AppCompatTextView {
        return toolbar_title
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setSupportActionBar(toolbar)
        setContentView(R.layout.activity_title)

        if (savedInstanceState == null) {
            startFragment(AboutFragment(), R.id.fragment_view)
        }
    }
}